pkg/cfn/builder/nodegroup.go

```diff
@@ -274,6 +274,9 @@ func nodeGroupResource(launchTemplateName *gfn.Value, vpcZoneIdentifier interfac
 	if ng.MaxSize != nil {
 		ngProps["MaxSize"] = fmt.Sprintf("%d", *ng.MaxSize)
 	}
+	if len(ng.ASGMetricsCollection) > 0 {
+		ngProps["MetricsCollection"] = ng.ASGMetricsCollection
```

Comment:
I think we shouldn't couple the API schema of eksctl with the cloudformation template directly. It could happen in the future that the schema is extended or changed to a new version and that would break the backend part of eksctl.

Even though it's verbose I think the right thing to do is to recreate the CFN struct like we do in line 287 with `mixedInstancesPolicy()`

Reply:
@martina-if , can you please verify the latest changes?